ChecksFinderClient.py

```diff
@@ -0,0 +1,703 @@
+from __future__ import annotations
```

Comment:
This entire file seems to be a modified copy of CommonClient.py
Copied code should be avoided where possible, please use imports. See FactorioClient or FF1Client as examples.

Reply:
I had to keep the server_loop clone because it was overwriting my custom process_server_cmd if I import it for some reason